test/python/transpiler/test_xy4.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2018.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Tests XY4 DD Pass"""
+
+from qiskit import QuantumCircuit
+from qiskit.transpiler import PassManager
+from qiskit.transpiler.passmanager_config import PassManagerConfig
+from qiskit.transpiler.passes import XY4Pass
+from qiskit.test import QiskitTestCase
+from qiskit.test.mock import FakeAlmaden
+
+class TestXY4(QiskitTestCase):
+    """Test the XY4 DD pass."""
+
+    def setUp(self):
+        self.backend = FakeAlmaden()
+        self.dt_in_sec = self.backend.configuration().dt
+        self.backend_prop = self.backend.properties()
+
+    def test_xy4_simple(self):
+        """Test the XY4 DD pass.
+
+        It should replace large enough delay blocks with XY4 DD sequences.
+        """
```

Comment:
```suggestion
        """Test that the pass replaces large enough delay blocks with XY4 DD sequences."""
```
The first line is already implied by the class. Please do this for the other two docstrings:)